spokestack/profile/wakeword.py

```diff
@@ -0,0 +1,56 @@
+"""
+Pipeline profile for pyaudio input, vad, wakeword.
+"""
+from typing import Any
+
+from spokestack.activation_timeout import ActivationTimeout
+from spokestack.agc.webrtc import AutomaticGainControl
+from spokestack.io.pyaudio import PyAudioInput
+from spokestack.nsx.webrtc import AutomaticNoiseSuppression
+from spokestack.pipeline import SpeechPipeline
+from spokestack.vad.webrtc import VoiceActivityDetector
+from spokestack.wakeword.tflite import WakewordTrigger
+
+
+class SpokestackWakeword:
+    """ Spokestack wakeword profile. """
+
+    @staticmethod
+    def create(
+        sample_rate: int = 16000,
+        frame_width: int = 20,
+        model_dir: str = "",
```

Comment:
this profile will fail immediately if `model_dir` isn't specified, so it shouldn't be an optional parameter